WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -1691,9 +1694,19 @@ static GSourceFuncs _handlerIntervention =
             }
 
             // Memory Pressure
-            if (_config.MemoryPressure.Value().empty() == false) {
-                Core::SystemInfo::SetEnvironment(_T("WPE_POLL_MAX_MEMORY"), _config.MemoryPressure.Value(), !environmentOverride);
+
+#if !(WEBKIT_CHECK_VERSION(2, 38, 0))
```

Comment:
Better to check if the version is less than 2.38. Otherwise this will break for later versions